confluence/confluence-v1.asciidoc

```diff
@@ -1,77 +1,99 @@
+|/wiki/rest/api/content|GET    |type=blogpost&start=0&limit=1&expand=body.storage|Returns all blogposts from the specified endpoint.|Root Request
+|/wiki/rest/api/content/${LW_PARENT_DATA_KEY}/child/comment|GET|expand=body.storage|Returns all comments in the blogposts from the specified endpoint. The value of `id` from the main request needs to be assigned to the `${LW_PARENT_DATA_KEY}` variable so the additional feature can insert that value when building the GET URL.  |Child Request
+|/wiki/rest/api/content | GET |type=paget&start=0&limit=1&expand=body.storage |Returns all pages from the specified endpoint.|Root Request
+|/wiki/rest/api/content/${LW_PARENT_DATA_KEY}/child/comment|GET|expand=body.storage|Returns all page comments from the specified endpoint. The value of `id` from the main request needs to be assigned to the `${LW_PARENT_DATA_KEY}` variable so the additional feature can insert that value when building the GET URL.|Child Request
 |=======================
 
 
+*Terminology*
+-------------
 
-Terminology
-~~~~~~~~~~~~
+The following terms are provided as a reference.
 
-* `Service Endpoints` - List of Service Endpoints
-* `Data Request` - Configure the endpoint request
-* `Child Endpoint` Request - The requests to retrieve additional information for the root data objects. It iterates over root data objects, and perform a request per each one.
-* `Root Response Mapping` - Define the mapping between the response and data objects to be indexed.
-* `Data Path` - The path to access a specific data object within a response. For example, in order to access a list of elements named with key 'objects', the DataPath would be 'objects'. If not provided, the entire response body will be indexed.
-* `DATA ID` - The identifier key for the data object, which value will be the solr-document's ID. If not provided, a random UUID will be used
-
-* `Root Response Mapping` - Define the mapping between the response and data objects to be indexed.
-* `Parent Data Key` - Key to extract data from root/parent response for subsequent request. The extracted value will be used to replace the ${LW_PARENT_DATA_KEY} variable in the child request configuration (endpoint, query params or body).For example, endpoint: /api/path/${LW_PARENT_DATA_KEY}/additionalInfo
-
-* `Child Data Path` - The path to access a specific object within a child response. For example, to access a list of elements named with key 'objects', the ChildDataPath would be 'objects'.If not provided, the entire response body will be indexed.
+[options="header"]
+|=======================
 
-* `Child Data ID` - The identifier key for the child data object, which value will be the solr-document's ID. Fill this when 'Custom Solr Field' is empty, otherwise the solr-document's ID will be a random UUID.
+|Term|Description
+|`Service Endpoints`|The list of service endpoints where data is retrieved. Each service endpoint configures a root endpoint request.
```

Comment:
The list of service endpoints from which the data is retrieved. Each service endpoint configures a root endpoint request.

Reply:
Changed.